src/app/clusters/identify-server/identify-server.cpp

```diff
@@ -199,38 +194,41 @@ void MatterIdentifyClusterServerAttributeChangedCallback(const app::ConcreteAttr
     }
 }
 
-bool emberAfIdentifyClusterIdentifyCallback(CommandHandler * commandObj, const app::ConcreteCommandPath & commandPath,
-                                            const Commands::Identify::DecodableType & commandData)
+bool emberAfIdentifyClusterIdentifyCallback(chip::app::CommandHandler * commandObj,
+                                            const chip::app::ConcreteCommandPath & commandPath,
+                                            const chip::app::Clusters::Identify::Commands::Identify::DecodableType & commandData)
```

Comment:
Since some of the old function names and signatures were wrong I copied these from callback.h. But some of those namespaces need removing. Will be fixed.